src/modules/Gapi.ts

```diff
@@ -0,0 +1,158 @@
+import { ISearchResultData } from "src/common/interface";
+
+declare global {
+  interface Window {
+    gapi: any; 
+  }
+}
```

Comment:
Do we still need this?

Reply:
If we remove this, the following error gets thrown:

`Property 'gapi' does not exist on type 'Window & typeof globalThis'.`